src/common/api/wsObjectApi.ts

```diff
@@ -0,0 +1,35 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
```

Comment:
need to refactor this file, right now it wont work because it isnt hooked up to state (as it isnt consuming the baseAPI)